config/config.py

```diff
@@ -1,3 +1,4 @@
 import json
 
+"""Holds flavor text"""
```

Comment:
I think it's more than just flavor text, since it's essentially almost all the text strings fed back to the user

Reply:
Changed to `Text that is fed to user as they navigate the application`

Reply:
That's a bit wordy, but I can't think of a better description